x-pack/platform/plugins/shared/ml/common/types/saved_objects.ts

```diff
@@ -39,6 +39,10 @@ export interface CanDeleteMLSpaceAwareItemsResponse {
   };
 }
 
+export interface CanSyncToAllSpacesResponse {
```

Comment:
We have a list of capabilities, e.g. `canCreateJob`. Shall we just include this one in the list?